time timestamp
string orderID
string item
string city
